### ISSUE TITLE
False negative: unresolved symbol with the same name as a namespace alias

### ISSUE THREAD

**Issue by yuhan0** [borkdude/clj-kondo#374]
**version**
clj-kondo v2019.07.24-alpha

**platform**
MacOS 10.14

**problem**

clj-kondo doesn't detect unresolved symbols when they happen to share the same name as a namespace alias.

**repro**
```clojure
(ns my.ns
  (:require [clojure.string :as string]))

(string/split string #" ")
```
**expected behavior**

The symbol `string` on the last line is an undefined var and results in a compile time error, it should be linted with a "unresolved symbol" warning.

**actual behavior**

No lint warnings produced.

**Comment by borkdude** [comment]
@yuhan0 Can be tested with https://4942-176829714-gh.circle-artifacts.com/0/release/clj-kondo-2019.07.25-alpha-SNAPSHOT-macos-amd64.zip

**Comment by borkdude** [comment]
Fixed with https://github.com/borkdude/clj-kondo/commit/18d97a6c3ca7d5d9eba351aa4291fc828bd2b215
